### DISCUSSION TITLE
How to integration lua custom plugin with kong manage

### DISCUSSION THREAD

**Discussion by vietduc030496** [Kong/kong#12831] [Help]
I use docker to run kong manage by tutorial: https://docs.konghq.com/gateway/latest/install/docker/#install-kong-gateway-with-a-database

Currently. i want to add simple custom plugin https://github.com/Kong/kong-plugin to kong-manage with above docker, but i don't know how to integration. I was read many document but i still don't understand and document is out of date

Can u help me step by step how to integration lua custom plugin with kong manage
Thank you

**Comment by Tieske** [comment]
You can try this: https://github.com/Kong/docker-kong/tree/master/customize
